WordPressKit/WordPressComOAuthClient.swift

```diff
@@ -125,13 +212,15 @@ public final class WordPressComOAuthClient: NSObject {
     ///     - success: block to be called if authentication was successful. The OAuth2 token is passed as a parameter.
     ///     - failure: block to be called if authentication failed. The error object is passed as a parameter.
     @available(*, deprecated, message: "Here for legacy compatiblity. Use the verions with explicit `needsMultifactor` parameter.")
-    @objc public func authenticateWithUsername(_ username: String,
-                                               password: String,
-                                               multifactorCode: String?,
-                                               success: @escaping (_ authToken: String?) -> Void,
-                                               failure: @escaping (_ error: NSError) -> Void ) {
-        authenticateWithUsername(
-            username,
+    public func authenticate(
+        username: String,
```

Comment:
I appreciate this and all the following Swift API syntax improvements.

From the PR description:

>  all the changes I'm about to make to WordPressKit and WordPressAuthenticator will be Swift first. A Swift friendly API is the first priority. If the new API can be bridged to Objective-C, then great. If it can't, I'll either create a Objective-C friendly wrapper or (at the worst case) translate the Objective-C callers to Swift.

100%. It's about time.